SR2019/2019-05-28-single-item-kit-chase.md

```diff
@@ -0,0 +1,23 @@
+---
+to: Teams with outstanding single items of kit
+subject: Outstanding kit
+---
+
+Hello,
+
+According to our records, you still have some of our kit. As the competition has ended, we would like this back!
```

Comment:
nit: maybe phrase like this?
```suggestion
According to our records, you still have some items from the robot kit you borrowed during the SR2019 competition. As the competition has ended, we would like this back!
```
This provides a bit more context and includes a reminder upfront that the kit was borrowed.